spec/unit/navigation/navigation.spec.js

```diff
@@ -7,6 +7,25 @@ const accordion = require("../../../src/js/components/accordion");
 
 const TEMPLATE = fs.readFileSync(path.join(__dirname, "template.html"));
 
+const EVENTS = {
```

Comment:
Semi-related to my previous comment, I notice the tests have a lot of these event dispatching helpers. Have y'all considered using a library like [Testing Library](https://testing-library.com/) to simplify this? In my experience, it also helps account for a lot of non-obvious [real-world](https://testing-library.com/docs/guiding-principles) details of events, like how clicking an element would also cause a focus event to be fired.

e.g.

```js
import userEvent from '@testing-library/user-event';

// ...

it("collapses dropdowns when the Escape key is hit", () => {
  userEvent.click(accordionButton);
  userEvent.tab();
  userEvent.keyboard('{escape}');
  assert.strictEqual(document.activeElement, accordionButton);
  assert.strictEqual(accordionButton.getAttribute("aria-expanded"), "false");
});
```

Reply:
This would definitely be useful and help bring consistency to the codebase. Thanks for the suggestion.

We still have to take a good look at the unit testing library and see if it's still up to par with what's needed. We've talked about incorporating cypress in the past.